src/sinks/redis/integration_tests.rs

```diff
@@ -287,6 +297,85 @@ async fn redis_sink_list_rpush() {
     }
 }
 
+#[tokio::test]
+async fn redis_sink_sorted_set_zadd() {
+    trace_init();
+
+    let key = Template::try_from(format!("test-{}", random_string(10)))
+        .expect("should not fail to create key template");
+    debug!("Test key name: {}.", key);
+    let mut rng = rand::rng();
+    let num_events = rng.random_range(10000..20000);
+    debug!("Test events num: {}.", num_events);
+
+    let cnf = RedisSinkConfig {
+        endpoint: OneOrMany::One(redis_server()),
+        key: key.clone(),
+        encoding: JsonSerializerConfig::default().into(),
+        data_type: DataTypeConfig::SortedSet,
+        list_option: None,
+        sorted_set_option: Some(SortedSetOption {
+            method: SortedSetMethod::ZAdd,
+        }),
+        score: Some(UnsignedIntTemplate::try_from("{{ num }}").unwrap()),
+        batch: BatchConfig::default(),
+        request: TowerRequestConfig {
+            rate_limit_num: u64::MAX,
+            ..Default::default()
+        },
+        sentinel_service: None,
+        sentinel_connect: None,
+        acknowledgements: Default::default(),
+    };
+
+    let mut events: Vec<Event> = Vec::new();
+    for i in 0..num_events {
+        let s: String = i.to_string();
+        let mut e = LogEvent::from(s);
+        e.insert("num", i);
+        events.push(e.into());
+    }
+    let input = stream::iter(events.clone().into_iter().map(Into::into));
+
+    // Publish events.
+    let cnf2 = cnf.clone();
+    assert_sink_compliance(&SINK_TAGS, async move {
+        // let conn = cnf2.build_connection().await.unwrap().get_connection_manager().await.unwrap().connection;
+        let cx = SinkContext::default();
+        let (sink, _healthcheck) = cnf2.build(cx).await.unwrap();
+        sink.run(input).await
+    })
+    .await
+    .expect("Running sink failed");
+
+    let mut conn = cnf
+        .build_connection()
+        .await
+        .unwrap()
+        .get_connection_manager()
+        .await
+        .unwrap()
+        .connection;
+
+    let key_exists: bool = conn.exists(key.clone().to_string()).await.unwrap();
+    debug!("Test key: {} exists: {}.", key, key_exists);
+    assert!(key_exists);
+    let zcount: usize = conn
+        .zcount(key.clone().to_string(), 0, num_events - 1)
+        .await
+        .unwrap();
+    debug!("Test key: {} count: {}.", key, zcount);
```

Comment:
```suggestion
    debug!("Test key: {key} count: {zcount}.");
```

Same for all places where the parameter can be inlined.